lib/tools/apk-utils.js

```diff
@@ -53,19 +53,30 @@ apkUtilsMethods.isAppInstalled = async function isAppInstalled (pkg) {
  *
  * @param {string} uri - The name of URI to start.
  * @param {string} pkg - The name of the package to start the URI with.
+ * @param {boolean} waitForLaunch - Optional parameter of waitForLaunch caps to add wait in adb command
```

Comment:
please make the third parameter an object: `opts = {}`. then `waitForLaunch` could be one of its properties

Reply:
This how it is going to look.
apkUtilsMethods.startUri = async function startUri (uri, pkg, opts = { waitForLaunch: true }) {

Reply:
please also update docs.
Also, it would be more readable to destruct the `opts` arg inside the method body (especially when it has more possible properties). See https://github.com/appium/appium-adb/blob/d8fce0016c3d532f251f2ae6b1351993dd6cbb83/lib/tools/settings-client-commands.js#L46 for more details

Reply:
https://github.com/appium/appium-adb/blob/master/docs/COMMANDS.md
Here I need to update correct with two scenario with false or without.